bin/nbb

```diff
@@ -344,6 +339,24 @@ find_centos_7_ami_ids() {
   done
 }
 
+shutdown_instances() {
+  aws ec2 describe-instances --filters "Name=instance-state-name,Values=running" | jq -c -r '.Reservations[] .Instances[] .InstanceId' > instance_list
+  echo "You are about to terminate the following instances:"
+  cat instance_list
+  read -r -p "Are you sure? [y/N] " response
+  if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]
+  then
+    IFS=$'\n'  # make newlines the separator
+    for i in `cat ./instance_list`; do
+      echo "Shutting down $i"
+      aws ec2 terminate-instances --instance-ids $i
```

Comment:
_specifically_, the file that gets called on the bastion host during a `make destroy` is put there during the `make provision-base` and is called here:  https://github.com/cloudfoundry-community/aws-nat-bastion-bosh-cf/blob/master/scripts/provision-bastion#L503 

After this is done (in theory) and whether or not that command succeeds, `make destroy` immediately calls terraform's destroy, which will fail if there are instances that it does not manage in the vpc that it does manage, but possibly not after it has destroyed the bastion host that it does manage.